algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp

```diff
@@ -123,18 +123,20 @@ struct TrivialTransformReduceBinaryTransformer {
   }
 };
 
-// put all code here and don't call from main
-// so that even if one runs the executable,
-// nothing is run anyway
-
-namespace KE     = Kokkos::Experimental;
-using count_type = std::size_t;
-using T          = double;
-Kokkos::View<T *> in1("in1", 10);
-Kokkos::View<T *> in2("in2", 10);
-Kokkos::View<T *> in3("in3", 10);
-Kokkos::DefaultExecutionSpace exe_space;
-std::string const label = "trivial";
+namespace KE = Kokkos::Experimental;
+
+struct TestStruct {
+  // put all code here and don't call from main
+  // so that even if one runs the executable,
+  // nothing is run anyway
+
+  using count_type      = std::size_t;
+  using T               = double;
+  Kokkos::View<T *> in1 = Kokkos::View<T *>("in1", 10);
+  Kokkos::View<T *> in2 = Kokkos::View<T *>("in2", 10);
+  Kokkos::View<T *> in3 = Kokkos::View<T *>("in3", 10);
```

Comment:
```suggestion
  Kokkos::View<T *> in1("in1", 10);
  Kokkos::View<T *> in2("in2", 10);
  Kokkos::View<T *> in3("in3", 10);
```

Reply:
That doesn't work
```
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:135:25: error: expected parameter declarator
  Kokkos::View<T *> in1("in1", 10);
                        ^
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:135:25: error: expected ')'
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:135:24: note: to match this '('
  Kokkos::View<T *> in1("in1", 10);
                       ^
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:136:25: error: expected parameter declarator
  Kokkos::View<T *> in2("in2", 10);
                        ^
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:136:25: error: expected ')'
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:136:24: note: to match this '('
  Kokkos::View<T *> in2("in2", 10);
                       ^
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:137:25: error: expected parameter declarator
  Kokkos::View<T *> in3("in3", 10);
                        ^
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:137:25: error: expected ')'
/tmp/kokkos/algorithms/unit_tests/TestStdAlgorithmsCompileOnly.cpp:137:24: note: to match this '('
  Kokkos::View<T *> in3("in3", 10);
```

Reply:
I didn't pay attention that these were class-member initializer.  `= {...};` would work.